### ISSUE TITLE
Not redirected to login when JWT has expired

### ISSUE THREAD

**Issue by heyheyjp** [LearnersGuild/echo-chat#19]
I hit `echo-chat` (locally) and attempted a command. I expected an error response, but not one that was related to authentication.

Received the message:

> **Error:** You are not authorized to do that.. Try `--help` for usage.

The `game` server had logged the following:

> `Error getting user from JWT: jwt expired`

Logging out and back in resolved the problem, but if the token was expired, I should have been redirected to login again as soon as I attempted to load the chat app.

We have two issues:
1. The error messaging in the app was not at all helpful. It should have told me not that I wasn't allowed to do something but that I needed to log in again.
2. It should have forced me to log in again _before_ loading the app. :)

**Comment by heyheyjp** [comment]
@jeffreywescott: if this is an issue that'll affect the players in prod, we should fix it before launching. If I hadn't been able to look at the server log (as they obvs won't), I would have felt really confused and stuck.

**Comment by jeffreywescott** [comment]
Have you seen this in production?

I thought I had squashed this one before (when I implemented sliding sessions in echo-chat). It may not happen in production because in production the login tokens for Meteor are set to the same duration as the JWT (24h), but the default is like 30d (IIRC).

To make this 1000% solid, we need to see if there's a way (when someone is allowed in via their Meteor "resume token") to have it _also_ check the stored JWT, and if the JWT is expired, it should boot them out. I'm not exactly where in Meteor the "login via resume token" stuff gets called, and / or how to hook into it.
